bluetooth/bluetooth-control

```diff
@@ -0,0 +1,160 @@
+  else
+    printf "The device is already discoverable. \n"
+  fi
+fi
+
+if [ -a /var/cache/bluetooth/reconnect_device ]; then
+
+# Get the alsa name of the connected device
+DEVICE="$(echo "$(amixer -D bluealsa scontrols)" | sed -n " s,[^']*'\([^']*\).*,\1,p ")"
+
+# Get and parse the MAC Adress of the connected device
+MAC="$(cat /var/cache/bluetooth/reconnect_device)"
+MAC_PARSED="$(echo ${MAC//:/_})"
+# PLAYER_ADDRESS="$(echo "$(dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED org.freedesktop.DBus.Properties.Get string:org.bluez.MediaControl1 string:Player)" | cut -d'"' -f 2)"
+
+if [[ ! -z "$(echo "$(amixer -D bluealsa scontrols)" | sed -n " s,[^']*'\([^']*\).*,\1,p ")" ]]; then
+
+# Volume up
+if [[ "$(cat /sys/class/gpio/gpio$VOLUME_UP_GPIO/value)" == 1 ]]; then
+  printf "Setting volume higher. \n"
+  amixer -D bluealsa sset "$DEVICE" 10%+
+  sleep 0.5
+fi
+
+if [[ "$TIMER" == 1 ]]; then
+  if [[ "$(dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.freedesktop.DBus.Properties.Get string:org.bluez.MediaPlayer1 string:Status | cut -d'"' -f 2)" == *"playing"* ]]; then 
+    printf "Stoping music playback. \n"
+    dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Pause
+  else
+    printf "Starting music playback. \n"
+    dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Play
+  fi
+fi
+
+if [[ "$TIMER" == 2 ]]; then
+  if [[ "$(dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.freedesktop.DBus.Properties.Get string:org.bluez.MediaPlayer1 string:Status | cut -d'"' -f 2)" == *"playing"* ]]; then
+    printf "Stoping music playback. \n"
+    dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Pause
+  else
+    printf "Starting music playback. \n"
+    dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Play
+  fi
+fi
+
+# Volume down
+if [[ "$(cat /sys/class/gpio/gpio$VOLUME_DOWN_GPIO/value)" == 1 ]]; then
+  printf "Setting volume lower. \n"
+  amixer -D bluealsa sset "$DEVICE" 10%-
+  sleep 0.5
+fi
+
+# Next track
+while [[ "$(cat /sys/class/gpio/gpio$NEXT_GPIO/value)" == 1 ]]; do
+  if [[ "$SKIP_STOP" == 0 ]]; then
+  printf "Playing the next track. \n"
+  dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Next
+  sleep 0.5
+  fi
+  SKIP_STOP="1"
+done
+
+SKIP_STOP="0"
+
+# Previous track
+while [[ "$(cat /sys/class/gpio/gpio$PREVIOUS_GPIO/value)" == 1 ]]; do
+  if [[ "$SKIP_STOP" == 0 ]]; then
+  printf "Playing the previous track. \n"
+  dbus-send --system --dest=org.bluez --print-reply /org/bluez/hci0/dev_$MAC_PARSED/player0 org.bluez.MediaPlayer1.Previous
+  sleep 0.5
+  fi
+  SKIP_STOP="1"
+done
```

Comment:
Same as before, if the code is nearly identical we should try and use functions to reduce code (also easier maintenance!)